src/nvim/testdir/test_regexp_utf8.vim

```diff
@@ -121,6 +145,37 @@ func s:classes_test()
   call assert_equal("\t\n\x0b\f\r ", spacechars)
   call assert_equal("\t", tabchar)
   call assert_equal('0123456789ABCDEFabcdef', xdigitchars)
+
+  if has('win32')
```

Comment:
I wonder if this case is because of Vim's `utf-8` implementation.

Reply:
This is because the default for Windows `isident` is different from vim (vim: `@,48-57,_,128-167,224-235`, noevim: `@,48-57,_,192-255`). I looked at it in `git log -p src/nvim/{options.lua,option.c}` and it's not clear why. Maybe it should be the same as vim.

Reply:
It works